### DISCUSSION TITLE
On the unification of minerals

### DISCUSSION THREAD

**Discussion by TingYu258** [GalaxyOfZero/Skyfriends#27] [Ideas]
Hello!When I was playing this modpack, I found that one mineral can burn many kinds of ingots. I think a good modpack should not be like this. It should be like chroma sky 2. There is only one kind of ingot. When firing a mineral with any mod machine, only one kind of ingot can be obtained. I hope it can be improved in the next update. Thank you

**Comment by GalaxyOfZero** [comment]
first of it doesn't matter how many arts it give from one ingot and second it is a hell on work in inplement only one are of all ingots. but i make this hell on work. in the 2.0 Update it give only one are of ingots 
[Maybe a unified mineral dictionary?](https://github.com/GalaxyOfZero/Skyfriends/issues/19)
little info, im the only worker on the pack and it cost a lot on time for all of this, the 2.0 has so much changes

**Comment by TingYu258** [comment]
Thank you. I'm looking forward to the upcoming 2.0. Although I don't know when it will be officially released, I'm looking forward to it
